src/annotator/test/annotation-sync-test.js

```diff
@@ -45,9 +45,9 @@ describe('AnnotationSync', function() {
     sandbox.restore();
   });
 
-  describe('channel event handlers', function() {
-    describe('the "deleteAnnotation" event', function() {
-      it('broadcasts the "annotationDeleted" event over the local event bus', function() {
+  describe('#constructor', function() {
```

Comment:
Of course there is no method named `constructor()` here. `AnnotationSync` _is_ itself the constructor method. But I didn't want to put the constructor tests at top-level within the `describe()`, as I think that would be a bad example, encouraging people to generally put everything top-level within the `describe()` and not organize by method. People would not realize that it's only the _contructor_'s tests that were top-level and further down any other methods tests were grouped.

I didn't want a `describe('#AnnotationSync'` within the `describe('AnnotationSync'` either, that seems weird.

I went with `describe('#constructor'` because `constructor()` is what the constructor method is called in ES6 classes.